build-arch-gce

```diff
@@ -139,6 +139,11 @@ arch-chroot -- "$mount_dir" /bin/bash -s -- "$loop_dev" <<-'EOS'
 		google-accounts-daemon google-clock-skew-daemon google-instance-setup \
 		google-network-daemon google-shutdown-scripts google-startup-scripts
 
+	echo "-- setting mtu"
+	cat <<-'EOF' > /etc/udev/rules.d/10-network.rules
+		ACTION=="add", SUBSYSTEM=="net", KERNEL=="eth*", ATTR{mtu}="1460"
```

Comment:
On the GCE instance I have, the network interface is ens4, not ethX. I think "en*" should be sufficient here per https://github.com/systemd/systemd/blob/master/src/udev/udev-builtin-net_id.c#L13

```suggestion
		ACTION=="add", SUBSYSTEM=="net", KERNEL=="en*", ATTR{mtu}="1460"
```

Reply:
It is "ens4" on my instance, too.
And "eth*" and "en*" both work.
I have updated this to "en*".